js/src/checkbox.js

```diff
@@ -0,0 +1,12 @@
+ // Checkbox classes 
++function ($) {
+$(document).ready(function() {
+         $(".check-hide input:checked").parent().remove()
+         $(".check-strike input:checked").next().css( "text-decoration", "line-through" )
+});
+$(document).on('click',function() {
+         $(".check-hide input:checked").parent().fadeOut('slow')
```

Comment:
Also, this should look at the target of the click event rather than performing several global DOM queries.